CinnVIIStarkMenu@NikoKrause/po/CinnVIIStarkMenu.pot

```diff
@@ -108,11 +108,11 @@ msgid "Menu"
 msgstr ""
 
 #: 3.2/applet.js:3260 2.6/applet.js:3280
-msgid "This file is no longer available"
+msgid "This file can't be found"
```

Comment:
Not sure about this one. There can be for example a file on a USB Stick. If you unmount the stick the file is not available. Plug the stick in and the file is available again. Saying the file can't be found, could be confusing in this context?